Fix issue with batch size failing for single repo

Fix issue with batch size failing for single repo

Seeing error in workflow in the call to `repo-stats-plus`

```bash
file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:712
        throw new Error(`Invalid number: ${value}`);
              ^

Error: Invalid number: 
    at Option.parseIntOption [as parseArg] (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:712:15)
    at Command._callParseArg (/home/runner/.local/share/gh/extensions/gh-repo-stats-plus/node_modules/commander/lib/command.js:601:21)
    at handleOptionValue (/home/runner/.local/share/gh/extensions/gh-repo-stats-plus/node_modules/commander/lib/command.js:702:20)
    at Command.<anonymous> (/home/runner/.local/share/gh/extensions/gh-repo-stats-plus/node_modules/commander/lib/command.js:728:9)
    at Command.emit (node:events:508:20)
    at /home/runner/.local/share/gh/extensions/gh-repo-stats-plus/node_modules/commander/lib/command.js:1980:18
    at Array.forEach (<anonymous>)
    at Command._parseOptionsEnv (/home/runner/.local/share/gh/extensions/gh-repo-stats-plus/node_modules/commander/lib/command.js:1967:18)
    at Command._parseCommand (/home/runner/.local/share/gh/extensions/gh-repo-stats-plus/node_modules/commander/lib/command.js:1552:10)
    at /home/runner/.local/share/gh/extensions/gh-repo-stats-plus/node_modules/commander/lib/command.js:1367:27

Node.js v25.8.2
Error: Process completed with exit code 1.
```

--- a/action.yml
+++ b/action.yml
@@ -362,14 +362,20 @@ runs:
         if [[ -n "${BATCH_SIZE}" ]]; then
           [[ "${BATCH_SIZE}" =~ ^[0-9]+$ ]] && [[ "${BATCH_SIZE}" -ge 1 ]] || { echo "::error::batch-size must be a positive integer"; exit 1; }
           BATCH_ARGS+=(--batch-size "${BATCH_SIZE}")
+        else
+          unset BATCH_SIZE
         fi
         if [[ -n "${BATCH_INDEX}" ]]; then
           [[ "${BATCH_INDEX}" =~ ^[0-9]+$ ]] || { echo "::error::batch-index must be a non-negative integer"; exit 1; }
           BATCH_ARGS+=(--batch-index "${BATCH_INDEX}")
+        else
+          unset BATCH_INDEX
         fi
         if [[ -n "${BATCH_DELAY}" ]]; then
           [[ "${BATCH_DELAY}" =~ ^[0-9]+$ ]] || { echo "::error::batch-delay must be a non-negative integer"; exit 1; }
           BATCH_ARGS+=(--batch-delay "${BATCH_DELAY}")
+        else
+          unset BATCH_DELAY
         fi
 
         gh repo-stats-plus repo-stats \
@@ -397,14 +403,20 @@ runs:
         if [[ -n "${BATCH_SIZE}" ]]; then
           [[ "${BATCH_SIZE}" =~ ^[0-9]+$ ]] && [[ "${BATCH_SIZE}" -ge 1 ]] || { echo "::error::batch-size must be a positive integer"; exit 1; }
           BATCH_ARGS+=(--batch-size "${BATCH_SIZE}")
+        else
+          unset BATCH_SIZE
         fi
         if [[ -n "${BATCH_INDEX}" ]]; then
           [[ "${BATCH_INDEX}" =~ ^[0-9]+$ ]] || { echo "::error::batch-index must be a non-negative integer"; exit 1; }
           BATCH_ARGS+=(--batch-index "${BATCH_INDEX}")
+        else
+          unset BATCH_INDEX
         fi
         if [[ -n "${BATCH_DELAY}" ]]; then
           [[ "${BATCH_DELAY}" =~ ^[0-9]+$ ]] || { echo "::error::batch-delay must be a non-negative integer"; exit 1; }
           BATCH_ARGS+=(--batch-delay "${BATCH_DELAY}")
+        else
+          unset BATCH_DELAY
         fi
 
         gh repo-stats-plus project-stats \

